programs/x509/cert_req.c

```diff
@@ -197,6 +219,57 @@ int main(int argc, char *argv[])
             }
         } else if (strcmp(p, "subject_name") == 0) {
             opt.subject_name = q;
+        } else if (strcmp(p, "san") == 0) {
+            prev = NULL;
+
+            while (q != NULL) {
+                if ((r = strchr(q, ',')) != NULL) {
```

Comment:
This might be problematic, as x509 names that we parse are usually separated by commas: https://github.com/Mbed-TLS/mbedtls/blob/development/tests/data_files/Makefile#L49.

Reply:
Are you willing to change the separator here, to be able to pass names separated by commas? I can also change that in a later PR, once such names are added.

Reply:
Hmm. We have here names separated by commas : `san=URI:http://pki.example.com/,IP:127.1.1.0,DNS:example.com`

Where is the problem?

Reply:
So maybe `;` can be used here?